docs/REFERENCE.md

```diff
@@ -359,6 +408,18 @@ const fetchDataOrErrorResult = Result.from(async () => {
 });
 ```
 
+Note: If an unexpected error is thrown within the `resultFactory` function, it will be wrapped in an `UnknownError` and passed to the `mapErr` or `mapAnyErr` function if they are in the chain. If not handled, it will result in a rejected promise.
+
+```ts
+import { Result, UnknownError } from 'type-safe-errors';
+
+Result.from(() => {
+  throw new Error('Problem!');
+})
+  .mapErr(UnknownError, err => console.error(err.cause));
+
+```
+
```

Comment:
```ts
await Result.from(() => { throw new Error('Problem!'); })
  .promise()
  .catch(err => console.error(err.cause) );  // err instanceOf UnknownError
```

Reply:
I do not get what do you suggest here.
First, this example purpose was to how to use `mapErr(UnknownError, ...)` syntax (you removed it).
Second, in your example you log `err.cause` which in most cases will be empty, because the promise will be rejected with original error, not with `UnknownError`.